package.json

```diff
@@ -7,7 +7,7 @@
     "build": "NODE_ENV=production lerna run build",
     "lint": "npm run lint:styles && npm run lint:scripts",
     "lint:fix": "npm run lint:styles:fix && npm run lint:scripts:fix",
-    "lint:styles": "stylelint --ignore-path .gitignore packages",
+    "lint:styles": "stylelint --ignore-path .gitignore packages/components/**/*.{css,scss,js,jsx,ts,tsx}",
```

Comment:
instead of continuing to add to `ignoreFiles`, figured this was more predictable

Reply:
I think we should do `packages/**/*.{css,scss,js,jsx,ts,tsx}` in case we add other bundles - though this may not play nice with the new addition of `dist` into the project. We may have to explicitly exclude that one.